spec/controllers/school/devices_controller_spec.rb

```diff
@@ -43,7 +43,7 @@
 
         it 'renders can_order' do
```

Comment:
this needs to be updated as it no longer matches the expectation...

Reply:
fixed up now